csvcubed/csvcubed/models/inspectdataframeresults.py

```diff
@@ -0,0 +1,58 @@
+"""
+Inspect Dataframe query results
+-------------------------------
+"""
+
+from os import linesep
+from typing import Optional
+import pandas as pd
+from dataclasses import dataclass
+
+from csvcubed.utils.printable import get_printable_tabuler_str_from_dataframe
+
+
+@dataclass
+class DatasetObservationsInfoResult:
+    """
+    Model to represent get dataset observation info dataframe operation result.
+    """
+
+    num_of_observations: int
+    num_of_duplicates: int
+    dataset_head: pd.DataFrame
+    dataset_tail: pd.DataFrame
+
+    @property
+    def output_str(self) -> str:
+        formatted_dataset_head = get_printable_tabuler_str_from_dataframe(
+            self.dataset_head
+        )
+        formatted_dataset_tail = get_printable_tabuler_str_from_dataframe(
+            self.dataset_tail
+        )
+        return f"{linesep}\t- Number of Observations: {self.num_of_observations}{linesep}\t- Number of Duplicates: {self.num_of_duplicates}{linesep}\t- First 10 Observations:{linesep}{formatted_dataset_head}{linesep}{linesep}\t- Last 10 Observations:{linesep}{formatted_dataset_tail}{linesep}"
```

Comment:
Can you split this into multiple strings so it's a bit easier to view?

Incidentally, have you got the black formatter installed - and are you running it on these files? It should auto-break the lines for you.

Reply:
https://pypi.org/project/black/

Reply:
@robons As per the attached image, I have black set up and it does not recognise this line as an unformatted line. But you are correct on the readability - I have done an update.

<img width="1280" alt="Screenshot 2022-04-04 at 13 09 57" src="https://user-images.githubusercontent.com/99191124/161540752-487ea129-1195-4553-9f61-b50799b23ebb.png">